pkg/config/config.go

```diff
@@ -647,6 +647,7 @@ func InitConfig(config Config) {
 	}
 
 	config.BindEnv("process_config.process_dd_url", "") //nolint:errcheck
+	config.SetDefault("process_config.enable_dns_by_querytype", false)
```

Comment:
I don't love that this is part of `process_config`, but since it is a temporary config option, I think it is OK.